octobot_trading/exchanges/parser/orders_parser_ccxt.py

```diff
@@ -0,0 +1,47 @@
+import octobot_trading.enums as enums
+import octobot_trading.exchanges.parser.orders_parser as orders_parser
+
+
+class CCXTOrdersParser(orders_parser.OrdersParser):
+    """
+    override CCXTOrdersParser if you add support for a CCXT exchange
+    dont use GenericCCXTOrdersParser as a base
+
+    parser usage:   parser = CCXTOrdersParser(exchange)
+                    orders = await parser.parse_orders(raw_orders)
+                    order = await parser.parse_order(raw_order)
+    """
+
+    TIMESTAMP_KEYS: list = [enums.ExchangeOrderCCXTColumns.TIMESTAMP.value]
+    STATUS_KEYS: list = [enums.ExchangeOrderCCXTColumns.STATUS.value]
+    ID_KEYS: list = [enums.ExchangeOrderCCXTColumns.ID.value]
+    SYMBOL_KEYS: list = [enums.ExchangeOrderCCXTColumns.SYMBOL.value]
+    SIDE_KEYS: list = [enums.ExchangeOrderCCXTColumns.SIDE.value]
+    TYPE_KEYS: list = [enums.ExchangeOrderCCXTColumns.TYPE.value]
+    TAKER_OR_MAKER_KEYS: list = [enums.ExchangeOrderCCXTColumns.TAKER_OR_MAKER.value]
+    PRICE_KEYS: list = [
+        enums.ExchangeOrderCCXTColumns.AVERAGE.value,  # first try average as its more accurate
+        enums.ExchangeOrderCCXTColumns.PRICE.value,
+    ]
+    FILLED_PRICE_KEYS: list = [enums.ExchangeOrderCCXTColumns.AVERAGE.value]
+    AMOUNT_KEYS: list = [enums.ExchangeOrderCCXTColumns.AMOUNT.value]
+    REMAINING_KEYS: list = [enums.ExchangeOrderCCXTColumns.REMAINING.value]
+    FILLED_AMOUNT_KEYS: list = []  # todo
```

Comment:
Shouldn't it be the same as AMOUNT_KEYS?

Reply:
Its actually ExchangeOrderCCXTColumns.FILLED, I changed it